care/emr/reports/context_builder/tag.py

```
@@ -0,0 +1,24 @@
+from care.emr.models import TagConfig
+from care.emr.reports.context_builder import Field, QuerysetContextBuilder
+
+
+class TagFilter(filters.FilterSet):
+    category = filters.CharFilter(field_name="category")
+    status = filters.CharFilter(field_name="status")
+
+
+class QuerysetTagContextBuilder(QuerysetContextBuilder):
+    filterset_class = TagFilter
+    __filterset_backends__ = [filters.DjangoFilterBackend]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)